assets/sass/protocols/_quiz1.scss

```diff
@@ -0,0 +1,74 @@
+@use '../components/icons';
+
+/* Mooc-grader questionnaire */
+
+.exercise .aplus-quiz1 {
+	.question-area {
+		margin-top: 20px;
+		margin-bottom: 20px;
+	}
+
+	.question-description {
+		margin-bottom: 5px;
+	}
+
+	.question-feedback {
+		margin-top: 5px;
+	}
+
+	.question-hint {
+		display: block;
+		margin-top: 5px;
+		margin-bottom: 5px;
+		color: #737373;
+		font-style: italic;
+	}
+
+	.hint-checkbox {
+		padding-left: 20px;
+	}
+
+	.option-checked {
+		font-weight: bold;
+	}
+
+	.error-text {
+		margin-top: 5px;
+		margin-bottom: 5px;
+		color: #a94442;
+		font-weight: bold;
+		margin-bottom: 0px;
+	}
+
+	.has-error .radio,
+	.has-error .checkbox {
+		color: #333;
+	}
+
+	.quiz1-icon-correct {
+		@include icons.glyphicon;
```

Comment:
I think it could be possible to use bootstrap3 classes here with [@extend](https://sass-lang.com/documentation/at-rules/extend). It could be possible to just write `@extend .glyphicon`.

Reply:
This would yield smaller CSS, but current solution is better base to start changing to font awesome.